src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlServiceImpl.java

```diff
@@ -127,7 +127,14 @@ public boolean canAccessCaseFieldsWithCriteria(final JsonNode caseFields,
     public boolean canAccessCaseViewFieldWithCriteria(final CommonField caseViewField,
                                                       final Set<AccessProfile> accessProfiles,
                                                       final Predicate<AccessControlList> criteria) {
-        return hasAccessControlList(accessProfiles, criteria, caseViewField.getAccessControlLists());
+        boolean hasAccess = hasAccessControlList(accessProfiles, criteria, caseViewField.getAccessControlLists());
+        if (!hasAccess) {
+            LOG.debug("No relevant case view field access for caseViewField={}, caseViewFieldAcls={}, userRoles={}",
```

Comment:
Should be error log and the error message can be improved

Reply:
Done. Will change this in the next commit

Reply:
It seems the change is not committed yet

Reply:
From looking around, I think the debug one should be left as is since no exception is thrown after the log message is displayed.